addons/mail/static/src/js/utils.js

```diff
@@ -146,7 +146,7 @@ function parseEmail(text) {
  */
 function escapeAndCompactTextContent(content) {
     //Removing unwanted extra spaces from message
-    let value = _.escape(content).trim();
+    let value = owl.utils.escape(content).trim();
```

Comment:
Isn't this necessary?
``` js
const { Component } = owl;
const { useRef } = owl.hooks;

```

Reply:
There is nothing like this in composer.js
https://github.com/odoo/odoo/blob/880555b38dee0c3e3b364c19bb6b0db983bf8aa0/addons/mail/static/src/models/composer/composer.js#L659

Plus, in odoo 14, `owl` available in console, so should be just a global variable

Reply:
Indeed no need to import keys that are not used, and `owl` is globally accessible. So current diff should be good.

However please add a test similar to other existing mention tests to cover this case.

Reply:
@seb-odoo it seems there is no mention tests. I added error log
```
        _generateMentionsLinks(body) {
            console.error("_generateMentionsLinks");
```
and only mail_to tour is failed, which doesn't check mentions

https://github.com/odoo/odoo/blob/82106350d3a7b82c6c529ae4ff9a2513ae34518e/addons/mail/static/src/js/tours/mail.js#L9

```

2020-12-08 10:35:42,077 1 ERROR test odoo.addons.test_mail.tests.test_ui: FAIL: TestUi.test_01_mail_tour
Traceback (most recent call last):
  File "/opt/odoo/auto/addons/test_mail/tests/test_ui.py", line 10, in test_01_mail_tour
    self.start_tour("/web", 'mail_tour', login="admin")
  File "/opt/odoo/custom/src/odoo/odoo/tests/common.py", line 1434, in start_tour
    res = self.browser_js(url_path=url_path, code=code, ready=ready, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/tests/common.py", line 1418, in browser_js
    self.fail('%s\n%s' % (message, error))
AssertionError: The test code "odoo.startTour('mail_tour')" failed
_generateMentionsLinks
 
```

Reply:
Check in

- odoo/addons/mail/static/src/components/composer/composer_tests.js
- odoo/addons/mail/static/src/components/composer_suggestion/composer_suggestion_partner_tests.js
- odoo/addons/mail/static/src/components/thread_view/thread_view_tests.js

Test should be
- setup with partner name with the problematic char
- user actions to mention, and then to send the message
- check the link on the posted message (due to this step, test will be in `thread_view_tests`)

Reply:
Thanks! I forgot about qunit tests

```
  -i web --test-tags=/web:WebSuite

```

Reply:
Done.

Before fix:

![image](https://user-images.githubusercontent.com/186131/101489808-c4b8fa00-3961-11eb-994d-4935d6478943.png)


After fix:

![image](https://user-images.githubusercontent.com/186131/101489741-a9e68580-3961-11eb-83bb-1993cace50fe.png)

Reply:
@seb-odoo do you want take a look before I merge?

Reply:
Yes please, or @alexkuhn